podwatcher/extension.go

```diff
@@ -0,0 +1,112 @@
+package podwatcher
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"runtime"
+
+	eirinix "code.cloudfoundry.org/eirinix"
+	"go.uber.org/zap"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+// https://github.com/cloudfoundry-incubator/eirini-staging/tree/master/image
+const (
+	defaultStagingExecutorEntrypoint   = "/packs/executor"
```

Comment:
It would then generate a cycle, as the constructor of the extension is setting explicitly defaults and would depend on `root.go`.  I would find that odd as `root.go` has the logic for the cli, but not of the extension itself... I do prefer to have the logic in one place, so I've exported the constants and attached them to the cli defaults in https://github.com/cloudfoundry-incubator/eirini-loggregator-bridge/pull/26/commits/7887894217194e764b8634c7b6857d4e9b915a9b